distributed/diagnostics/nvml.py

```diff
@@ -12,10 +13,17 @@
 nvmlOwnerPID = None
 
 
+def _in_wsl():
+    """Check if we are in Windows Subsystem for Linux; some PyNVML queries are not supported there.
+    Taken from https://www.scivision.dev/python-detect-wsl/
+    """
+    return "microsoft-standard" in uname().release
```

Comment:
What happens if this runs on non-WSL Windows (not that we support that just curious)?

Reply:
Looks like on standard Windows this just outputs the version of the Windows kernel; for example, on Windows 11:

```python
>>> import platform
>>> platform.uname().release
'10'
```